CefSharp.BrowserSubprocess.Core/JavascriptCallbackRegistry.h

```diff
@@ -0,0 +1,28 @@
+// Copyright � 2010-2014 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#pragma once
+
+#include "JavascriptCallbackWrapper.h"
+
+namespace CefSharp
+{
+    namespace Internals
+    {
+        ref class JavascriptCallbackRegistry
+        {
+        private:
+            int browserId;
+            Int64 lastId;
+            Dictionary<Int64, JavascriptCallbackWrapper^>^ callbacks = gcnew Dictionary<Int64, JavascriptCallbackWrapper^>();
```

Comment:
It's likely `callbacks` will need to be initialized in the constructor to avoid the current build errors. (Were you developing on `VS2013`? There are a few differences between that and `VS2012` in terms of `VC++`).